tests/api-testing/tests/test_pipeline.py

```
@@ -108,3 +108,237 @@ def test_pipeline_creation_and_action(create_session, base_url, pipeline_name, s
+
+
+
+@pytest.mark.parametrize(
+    "pipeline_name, stream_name, action, expected_status, schedule_enabled",
+    [
+        ("pipeline12222", "stream_pytest_data", None, 400, False),  # No action (default behavior)
+     
+    ]
+)
+
+def test_pipeline_creation_and_action_with_schedule(create_session, base_url, pipeline_name, stream_name, action, expected_status, schedule_enabled):
```

Ruff (0.8.2)

Redefinition of unused `test_pipeline_creation_and_action_with_schedule` from line 122

(F811)